### ISSUE TITLE
Hyper paramter turning

### ISSUE THREAD

**Issue by peterdudfield** [openclimatefix/PVNet#127]

**Comment by Sukh-P** [comment]
[Kicked off some hyperparameter tuning](https://wandb.ai/openclimatefix/pvnet_hyperparameter_tuning), looking into:
- Changing parameters linked to the NWP encoders like altering the number of conv3d channels, number of out features
If there is time:
- Changing the weight decay/regularisation value
- Changing the number of embedding dimensions for the GSP encoding

**Comment by Sukh-P** [comment]
Completed a first round of hyperparameter tuning experiments with some modest improvements (relative 3% improvement in normalised MAE compared to the current PVNet configuration) were seen on the small experiments I ran when using larger models/encoders for the NWP parts, definitely not conclusive due to caveats mentioned in the doc but perhaps enough to revisit this in the future and run a test on a model trained for longer. 

See this [doc](https://docs.google.com/document/d/1-fZ-PIjwBuByJFIhRgUBXTR2lbbIKVn2ww2igQ0OaVU/edit?usp=sharing) for a full writeup of what the experiments were and results.
